nano/node/blockprocessor.cpp

```diff
@@ -460,3 +477,38 @@ nano::stat::detail nano::to_stat_detail (nano::block_source type)
 	debug_assert (value);
 	return value.value_or (nano::stat::detail{});
 }
+
+/*
+ * block_processor_config
+ */
+
+nano::block_processor_config::block_processor_config (const nano::network_constants & network_constants)
```

Comment:
We have unit tests that test setting of default values.
Technically, those should update too.
Although they way they are at moment, I am fairly certain they are incomplete anyway, because people consistently forget to update them.

Reply:
I forgot about that, I will write another test case.